File: Python/LabWork/Customer.py
class Customer:

    def __init__(self, fio, passport, product):
        self.fio = fio
        self.passport = passport
        self.product = product

    def __str__(self):
        return f"Основная информация о покупателе: ФИО: {self.fio} Паспорт: {self.passport} {self.product}"

File: Магистратура/1 курс 2 семестр/ Java for A/projectF/src/main/java/com/practice/project/Customer.java
package com.practice.project;

import org.hibernate.annotations.GenericGenerator;
import javax.persistence.*;

@Entity
@Table(name = "customer")
public class Customer
{
    @Id
    @GeneratedValue(generator = "increment")
    @GenericGenerator(name = "increment", strategy = "increment")
    @Column(name = "id_customer")
    public Long id;

    @Column (name = "fio")
    public String fio;

    @Column (name = "email")
    public String email;

    @Column (name = "phone")
    public String phone;


    public Customer(String fio, String email, String phone) {
        this.fio = fio;
        this.email = email;
        this.phone = phone;
    }
}
